[2025-07-07T13:47:16.028+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-07T13:47:16.068+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fin_cotacoes_bcb_classic.create_table_postgres scheduled__2024-01-20T00:00:00+00:00 [queued]>
[2025-07-07T13:47:16.078+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fin_cotacoes_bcb_classic.create_table_postgres scheduled__2024-01-20T00:00:00+00:00 [queued]>
[2025-07-07T13:47:16.078+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-07-07T13:47:16.091+0000] {taskinstance.py:2330} INFO - Executing <Task(SQLExecuteQueryOperator): create_table_postgres> on 2024-01-20 00:00:00+00:00
[2025-07-07T13:47:16.104+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=51395) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-07T13:47:16.103+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fin_cotacoes_bcb_classic', 'create_table_postgres', 'scheduled__2024-01-20T00:00:00+00:00', '--job-id', '1664', '--raw', '--subdir', 'DAGS_FOLDER/cotacoes.py', '--cfg-path', '/tmp/tmpneds97je']
[2025-07-07T13:47:16.104+0000] {standard_task_runner.py:63} INFO - Started process 51425 to run task
[2025-07-07T13:47:16.104+0000] {standard_task_runner.py:91} INFO - Job 1664: Subtask create_table_postgres
[2025-07-07T13:47:16.165+0000] {task_command.py:426} INFO - Running <TaskInstance: fin_cotacoes_bcb_classic.create_table_postgres scheduled__2024-01-20T00:00:00+00:00 [running]> on host bc96ca91d307
[2025-07-07T13:47:16.265+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fin_cotacoes_bcb_classic' AIRFLOW_CTX_TASK_ID='create_table_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-01-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-20T00:00:00+00:00'
[2025-07-07T13:47:16.267+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-07T13:47:16.267+0000] {sql.py:276} INFO - Executing: 
CREATE TABLE IF NOT EXISTS cotacoes (
    dt_fechamento DATE,
    cod_moeda TEXT,
    tipo_moeda TEXT,
    desc_moeda TEXT,
    taxa_compra REAL,
    taxa_venda REAL,
    paridade_compra REAL,
    paridade_venda REAL,
    data_processamento TIMESTAMP,
    CONSTRAINT table_pk PRIMARY KEY (dt_fechamento, cod_moeda)
);
[2025-07-07T13:47:16.276+0000] {base.py:84} INFO - Using connection ID '***-postgres' for task execution.
[2025-07-07T13:47:16.287+0000] {base.py:84} INFO - Using connection ID '***-postgres' for task execution.
[2025-07-07T13:47:16.298+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-07T13:47:16.298+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py", line 282, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 431, in run
    with self._create_autocommit_connection(autocommit) as conn:
  File "/usr/local/lib/python3.12/contextlib.py", line 137, in __enter__
    return next(self.gen)
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 551, in _create_autocommit_connection
    with closing(self.get_conn()) as conn:
                 ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/hooks/postgres.py", line 175, in get_conn
    self.conn = psycopg2.connect(**conn_args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.OperationalError: could not translate host name "host.docker.internal" to address: Name or service not known

[2025-07-07T13:47:16.310+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=fin_cotacoes_bcb_classic, task_id=create_table_postgres, run_id=scheduled__2024-01-20T00:00:00+00:00, execution_date=20240120T000000, start_date=20250707T134716, end_date=20250707T134716
[2025-07-07T13:47:16.321+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 1664 for task create_table_postgres (could not translate host name "host.docker.internal" to address: Name or service not known
; 51425)
[2025-07-07T13:47:16.358+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-07-07T13:47:16.387+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-07T13:47:16.389+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-07T15:03:09.206+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-07T15:03:09.340+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fin_cotacoes_bcb_classic.create_table_postgres scheduled__2024-01-20T00:00:00+00:00 [queued]>
[2025-07-07T15:03:09.395+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fin_cotacoes_bcb_classic.create_table_postgres scheduled__2024-01-20T00:00:00+00:00 [queued]>
[2025-07-07T15:03:09.395+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-07-07T15:03:09.442+0000] {taskinstance.py:2330} INFO - Executing <Task(SQLExecuteQueryOperator): create_table_postgres> on 2024-01-20 00:00:00+00:00
[2025-07-07T15:03:09.483+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=94852) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-07T15:03:09.484+0000] {standard_task_runner.py:63} INFO - Started process 94935 to run task
[2025-07-07T15:03:09.488+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fin_cotacoes_bcb_classic', 'create_table_postgres', 'scheduled__2024-01-20T00:00:00+00:00', '--job-id', '2819', '--raw', '--subdir', 'DAGS_FOLDER/cotacoes.py', '--cfg-path', '/tmp/tmpt4r6tlri']
[2025-07-07T15:03:09.489+0000] {standard_task_runner.py:91} INFO - Job 2819: Subtask create_table_postgres
[2025-07-07T15:03:09.707+0000] {task_command.py:426} INFO - Running <TaskInstance: fin_cotacoes_bcb_classic.create_table_postgres scheduled__2024-01-20T00:00:00+00:00 [running]> on host bc96ca91d307
[2025-07-07T15:03:09.981+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fin_cotacoes_bcb_classic' AIRFLOW_CTX_TASK_ID='create_table_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-01-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-20T00:00:00+00:00'
[2025-07-07T15:03:09.982+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-07T15:03:09.982+0000] {sql.py:276} INFO - Executing: 
CREATE TABLE IF NOT EXISTS cotacoes (
    dt_fechamento DATE,
    cod_moeda TEXT,
    tipo_moeda TEXT,
    desc_moeda TEXT,
    taxa_compra REAL,
    taxa_venda REAL,
    paridade_compra REAL,
    paridade_venda REAL,
    data_processamento TIMESTAMP,
    CONSTRAINT table_pk PRIMARY KEY (dt_fechamento, cod_moeda)
);
[2025-07-07T15:03:10.015+0000] {base.py:84} INFO - Using connection ID '***-postgres' for task execution.
[2025-07-07T15:03:10.048+0000] {base.py:84} INFO - Using connection ID '***-postgres' for task execution.
[2025-07-07T15:03:10.086+0000] {sql.py:487} INFO - Running statement: 
CREATE TABLE IF NOT EXISTS cotacoes (
    dt_fechamento DATE,
    cod_moeda TEXT,
    tipo_moeda TEXT,
    desc_moeda TEXT,
    taxa_compra REAL,
    taxa_venda REAL,
    paridade_compra REAL,
    paridade_venda REAL,
    data_processamento TIMESTAMP,
    CONSTRAINT table_pk PRIMARY KEY (dt_fechamento, cod_moeda)
);, parameters: None
[2025-07-07T15:03:10.125+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-07T15:03:10.156+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=fin_cotacoes_bcb_classic, task_id=create_table_postgres, run_id=scheduled__2024-01-20T00:00:00+00:00, execution_date=20240120T000000, start_date=20250707T150309, end_date=20250707T150310
[2025-07-07T15:03:10.273+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-07T15:03:10.318+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-07T15:03:10.332+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
